app/views/application/_sticky_sidebar.html.slim

```diff
@@ -16,6 +16,11 @@
       input class='form-control' id='activity_search' placeholder='Search by activity name...'
   = render 'activities/filter'
   .row.text-center
-    = link_to 'add an activity', new_activity_path, class: 'btn btn-success btn-lg fa fa-plus' 
+    = link_to 'add an activity', "#", class: 'btn btn-success btn-lg fa fa-plus', 'data-toggle' => "modal", 'data-target' => '#newModal'
```

Comment:
Should we load that modal like the other one... with remote: true? Now then we figured out how to do this? ;)
I just gave it a try and it works quite fine with ajax.